### PR TITLE
Created Comment Feature

### PR DESCRIPTION
**Pull request by jennygaoo**
practiced Post requests

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -26,22 +26,29 @@
...
+  
+  private List<String> comments;
 
   @Override
-  public void init() {
-    messages = new ArrayList<>();
-    messages.add("hello");
-    messages.add("how are you");
-    messages.add("good morning");
+  public void init(){
```
Comment (jgao23) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:33]: add space before bracket

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -26,22 +26,29 @@
...
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = getParameter(request, "text-input", "");
+    
+    //want to display the text as an ArrayList
+    comments.add(text);
+    response.setContentType("text/html;");
+    response.getWriter().println(comments);
+  }
+
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```
Comment (jgao23) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: can hardcode "" and remove defaultValue as a function param

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -43,10 +43,20 @@ function randomizeRecipe() {
...
-  console.log('Fetching a message.');
+async function getComments() {
+  console.log('Fetching the comments.');
  
   const response = await fetch('/data');
-  const message = await response.json();
-  document.getElementById('message').innerHTML = message;
+  const comments = await response.json();
+
+  const commentsEl = document.getElementById('comments');
```
Comment (andreahu12) [inline portfolio/src/main/webapp/script.js:52]: nit: `commentsEl` --> `commentsElement` for clarity? (Should be able to easily tell what it is by variable name)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -26,22 +26,36 @@
...
-    messages.add("good morning");
+    comments = new ArrayList();
   }
-  
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = getParameter(request, "text-input");
+    
+    //want to display the text as an ArrayList
```
Comment (jgao23) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:40]: add space after //

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -39,14 +39,24 @@ function randomizeRecipe() {
...
 
-  imageContainer.innerHTML = '';
+  imageContainer.innerHTML = "";
   imageContainer.appendChild(imgElement);
 }
 
-async function getMessage() {
-  console.log('Fetching a message.');
+async function getComments() {
+  console.log("Fetching the comments.");
```
Comment (jgao23) [inline portfolio/src/main/webapp/script.js:47]: probably can remove this log message if you're done debugging

**Pull request merged**